tests/drivers/watchdog/wdt_basic_api/testcase.yaml

```diff
@@ -169,3 +169,9 @@ tests:
     extra_configs:
       - CONFIG_PM=y
       - CONFIG_PM_DEVICE=y
+  drivers.watchdog.basic_api.dcache:
+    platform_allow:
```

Comment:
if you add a case for this you need add a filter see https://github.com/zephyrproject-rtos/zephyr/blob/c3190d34aa912d39cd082db822f6ebb332806a2b/tests/drivers/uart/uart_async_api/testcase.yaml#L66

also you need filter out other cases for rt700.